frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
+}
+
+/* the name of the fieldset */
+.defence-mechanism-legend {
+	padding: 0.3em;
 }
 
-.defence-mechanism > * {
-	padding: 0.75rem;
+/* the more button to expand the details */
+.defence-mechanism-summary {
+	width: fit-content;
+	padding: 0.5rem;
+	border: 0.125rem outset var(--control-config-border);
+	border-radius: 0.25rem;
 }
 
 .defence-mechanism > summary {
-	position: relative;
 	font-size: 1rem;
 }
 
-/*
- * Modified from https://www.w3schools.com/howto/howto_css_switch.asp
-*/
+/* remove the default triangle on the summary element */
+.defence-mechanism-fieldset details > summary {
+	list-style: none;
+}
 
-/* The switch - the box around the slider */
-.defence-mechanism label.switch {
+.defence-mechanism-fieldset details > summary::-webkit-details-marker {
+	display: none;
+}
+
+/* style for when details element is expanded */
+.defence-mechanism-fieldset details[open] > summary {
+	border-style: inset;
+}
```

Comment:
this is clever. I but despite the comment, it's a bit hard to figure out what it's doing without inspecting in the browser...

Again, just an aesthetic point, but I'd prefer a flat button (to be consistent with rest of app), and then to indicate open or closed I'd put the little triangle (that used to be on the `<summary>`) on the button

Reply:
personally I think this is clearer than the triangle, especially since the triangle is already on the parent component, and the difference between expanded and a collapsed triangle is only a few degrees of rotation which is not that easy to see.
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/6f3416be-e4ac-40b7-a90e-022f3ff62149)

Reply:
We've had a discussion on Teams for this and agreed it's cool as is